neural_speed/models/llama/llama_utils.cpp

```diff
@@ -65,7 +65,7 @@ void Llama::init(const char* path_model, model_context* ctx, int n_gpu_layer_, b
   auto& hparams = model.hparams;
   n_ff = hparams.ffn_hidden_size;
   fprintf(stderr, "%s: n_vocab    = %u\n", __func__, hparams.n_vocab);
-  fprintf(stderr, "%s: n_ctx      = %u\n", __func__, hparams.max_seq_len);
+  fprintf(stderr, "%s: n_ctx      = %u\n", __func__, lctx.n_ctx);
```

Comment:
what is the default value of  lctx.n_ctx ?

Reply:
https://github.com/intel/neural-speed/blob/main/neural_speed/models/model_utils/model_types.h#L307